### DISCUSSION TITLE
How to remove Big LobeHub Preloading logo?

### DISCUSSION THREAD

**Discussion by antorio** [lobehub/sd-webui-lobe-theme#352] [Q&A] [answered]
How to remove Big LobeHub Preloading logo? 
There are already LobeHub logo/elements everywhere, header, favicon, footer, links, top icons, etc..
Just that every time I open SD, it feels like I'm opening a "LobeHub application", not a "Stable Diffusion application"

..and sorry the "out of color" emoji logo is just not my taste for opening the day

**Comment by antorio** [comment, marked as answer]
don't bother. I got it.

**Reply by Zeime** [reply]
Hi I just installed this and was trying to figure how to change the favicon, the loading screen, pretty much everywhere where the Lobehub icon appears, Please help me out, Thanks !
